.mailmap

```diff
@@ -219,6 +221,7 @@ Michael Hirsch <scienceopen@noreply.github.com> michael <scienceopen@noreply.git
 Michael Hirsch <scienceopen@noreply.github.com> Michael Hirsch <scienceopen@users.noreply.github.com>
 Michael James Bedford <SunsetOrange@users.noreply.github.com> Michael <SunsetOrange@users.noreply.github.com>
 Mikhail Pak <mikhail.pak@tum.de> mp4096 <mikhail.pak@tum.de>
+Nathan Bell <wnbell@localhost> wnbell <wnbell@localhost>
```

Comment:
Looking at https://github.com/wnbell/personal - I think Nathan's address is wnbell@gmail.com

Reply:
Right, but I think we need explicit consent to add email addresses here.

Reply:
Agree about consent - I didn't look for valid email addresses on purpose.